### DISCUSSION TITLE
Feature request: Code output highlighting

### DISCUSSION THREAD

**Discussion by malcolmbarrett** [quarto-dev/quarto-cli#4897] [Feature Requests]
xaringan has a [nice feature where](https://slides.yihui.org/xaringan/#31) you can highlight code *output* (not just the code itself). This is useful in teaching when you want to draw attention to some aspect of the output

I guess an extension could handle this, but it seems like this would be good to have in Quarto proper

cc @cderv with whom I discussed this a while back

**Comment by gshotwell** [comment]
Here's a filter which does this, but I agree that it would be good to have in quarto proper. Code highlighting is used throughout the quarto documentation itself.

https://github.com/shafayetShafee/line-highlight#highlighting-output-line-numbers

**Comment by EllaKaye** [comment]
The line-highlighting extension looks neat, but only works for the `html` format. I'd love to see this work for `revealjs` as well. I've put in a request for that in the extension, but if this functionality is being considered for Quarto itself, then it would be great if it could work across all formats.
